Makefile

```diff
@@ -16,3 +16,8 @@ sars-cov-2-images:
 synthetic-images:
 	docker build --build-arg pathogen=${SYNTHETIC} -t ${DOCKER_IMAGE_URI_PATH}/synthetic-pipeline:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.psga-pipeline .
 	docker build --build-arg pathogen=${SYNTHETIC} -t ${DOCKER_IMAGE_URI_PATH}/fastqc:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.fastqc .
+
+bactopia-images:
+	docker build --progress=plain -t ${DOCKER_IMAGE_URI_PATH}/s-aureus-pipeline:${DOCKER_IMAGE_TAG} -f docker/Dockerfile.s-aureus-pipeline .
```

Comment:
Could you use the `Dockerfile.psga-pipeline` here instead, please?
You will need to pass the pathogen name. This will load the dependencies for that pathogen.
We want this docker file to be pathogen generic but still configurable.

Reply:
There are good reasons for inheriting from that docker image - it sets up AWS batch and ensures the repo installs anything else the pipeline expects to be generally accessible. On the basis this would be a lot of pain right now and we don't have the time I think we need to ticket to come back and do this?